doc/conf.py

```diff
@@ -28,9 +28,9 @@
 print("python exec:", sys.executable)
 print("sys.path:", sys.path)
 
-if "conda" in sys.executable:
+if "CONDA_DEFAULT_ENV" in os.environ or "conda" in sys.executable:
```

Comment:
The script might more reliably detect running within a *conda* environment by checking for the environment variable rather than looking for the text "conda" within the interpreter's path. For example, I installed Miniconda at a path of my choosing that happens to contain "conda" (only within the word "Miniconda") but did not need to. The value of the variable is the name of the environment that will be shown by `conda list` subsequently, but we don't need the name. I welcome comments.